src/Sanet.Transport.SignalR.Hub/Dockerfile

```
@@ -0,0 +1,46 @@
+# Use the official .NET 10 runtime as a parent image
+FROM mcr.microsoft.com/dotnet/aspnet:10.0 AS base
+WORKDIR /app
+EXPOSE 8080
+
+# Use the SDK image to build the application
+FROM mcr.microsoft.com/dotnet/sdk:10.0 AS build
+WORKDIR /src
+
+# Copy SDK pinning and shared build props so they are honored inside the container
+COPY ["global.json", "Directory.Build.props", "."]
```

Hadolint (2.15.1)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)